package/utils/busybox/Config-defaults.in

```diff
@@ -24,7 +24,7 @@ config BUSYBOX_DEFAULT_FEATURE_VERBOSE_USAGE
 	default y
 config BUSYBOX_DEFAULT_FEATURE_COMPRESS_USAGE
 	bool
-	default n
+	default y
```

Comment:
When the data is already compressed squashfs will not be able to compress it any more. This probably decreases the size of the busýbox binary, but it probably increases the size of the compressed root file system

Reply:
That's certainly a valid concern - I was working with the ext4 image type for x86 using qemu to test my results, and had not considered that (even though I use squashfs on all my "production" OpenWrt devices...). I will try both variants - compressed and uncompressed busybox help texts - with a squashfs image, and report the image size results later today. Even a little bit of data should be better than no data to decide whether or not that's a worthwhile option :)

Reply:
So I just built a defconfig squashfs images with all the proposed busybox changes (including compression of help texts) for x86/64/generic, and the resulting `openwrt-x86-64-generic-squashfs-rootfs.img.gz` measured *4098069* bytes.

Then, I repeat that build with identical configuration, except for *disabling* the `CONFIG_BUSYBOX_CONFIG_FEATURE_COMPRESS_USAGE` option, and the resulting `openwrt-x86-64-generic-squashfs-rootfs.img.gz` measured *4094607* bytes.

Ergo, enabling busybox help/usage text compression inside the gzipped squashfs image *increased* the size of that image by *3462* bytes.